Integrations/integration-fireeye.yml

```diff
@@ -414,8 +464,8 @@ script:
       description: Bad hash found
     - contextPath: File.Malicious.Vendor
       description: For malicious files, the vendor that made the decision
-    description: commands.server.feSubmit.arguments.submissionKey.description
-  - name: fe-submit
+    description: Results of the file submission
+  - name: fe-submit-file
```

Comment:
@sgosai I think it might be better to keep this as before for backwards compatibility

Reply:
@hirshbergshachar, integration has been updated.